### PR TITLE
fix bug with QDL and [art] detetion

### PR DESCRIPTION
**Pull request by toddmedema**

### CODE REVIEW

File: app/playtest/PlaytestCrawler.tsx [hunk 1]
```diff
@@ -77,6 +77,9 @@ export class PlaytestCrawler extends StatsCrawler {
 
   private verifyRoleplayArt(roleplayNode: Node<Context>, line: number) {
     roleplayNode.loopChildren((tag, child, orig) => {
+      if (tag === 'choice') {
+        return; // bug with loop children returning all children text dumped into a single line
```
Comment (smartin015) [inline app/playtest/PlaytestCrawler.tsx]: If this is a bug, is there a bug reference in the QDL repo? if so, link it here as a TODO. If not... suggest clarifying this (I'm not sure what this is fixing).

Comment (toddmedema) [reply app/playtest/PlaytestCrawler.tsx]: Added link to the bug; let me know if you have any thoughts on how to address it in a more elegant manner.

https://github.com/ExpeditionRPG/expedition-qdl/issues/63

Comment (smartin015) [reply app/playtest/PlaytestCrawler.tsx]: I'm not convinced this is a bug... see comments on the issue.

File: app/playtest/PlaytestCrawler.tsx [hunk 2]
```diff
@@ -78,7 +78,9 @@ export class PlaytestCrawler extends StatsCrawler {
   private verifyRoleplayArt(roleplayNode: Node<Context>, line: number) {
     roleplayNode.loopChildren((tag, child, orig) => {
       if (tag === 'choice') {
-        return; // bug with loop children returning all children text dumped into a single line
+        // bug with loop children returning all children text dumped into a single line
+        // https://github.com/ExpeditionRPG/expedition-qdl/issues/63
```
Comment (smartin015) [inline app/playtest/PlaytestCrawler.tsx]: See response to the issue... your PR is the correct thing to do, and it's not a bug that loopChildren does what it does. Let's remove these comments, then LGTM.

**Pull request merged**